cana/utils.py

```diff
@@ -64,6 +65,18 @@ def statenum_to_binstate(statenum, base):
 	# bstate = bin(statenum)[2:].zfill(base)
 	return bstate
 
+
+def statenum_to_output_list(statenum, base):
+	'''
```

Comment:
@xuan-w ,same comment as above. We have to use the same documentation style so sphinx will understand it for the Docs.